content/releases/4-0/1_design/snippet.php

```diff
@@ -36,7 +36,20 @@
 </style>
 
 <figure class="release-box bg-light mb-6">
-	<?= $section->image('company.png')->resize(1600) ?>
+	<?= img($section->image('company.png'), [
+		'src' => [
+			'width' => 1248
+		],
+		'lazy' => false,
+		'sizes' => '(min-width: 1540px) 1248px, (min-width: 1160px) calc(77.78vw + 66px), (min-width: 480px) calc(100vw - 96px), 90vw',
```

Comment:
`Comment:` These are by nature to be trusted (or using the same tool, which I just assume will generate the same results for me), but there is no easy natural way to derive at these values during review. I am wondering if we should add a comment where these are coming from for a few years down the road when we're wondering. Although it might be a bit annoying to add the comment everywhere. But can't think of a good central place.